### DISCUSSION TITLE
update some error messages

### DISCUSSION THREAD

**Discussion by amirmahdifard** [BearWare/TeamTalk5#1629] [Ideas]
### Description

please update some server error messages to be more helpful, this also will be good if later you want to add ban duration: when in account list we disable a user, if that user trys to logging, he will get invalid account error, but invalid account is not true, may be should say disabled account or something like that: and also, you are so hard and you are not adding ban duration feecher, so at least add a box so when we banned someone, and we say the reezen in there, so when the person tryes to logging, he could get something like, you are permenantly banned from the server for following reezen: spamming the server and annoying people: and if you decide to add ban duration, they can get: you are banned from the server for 14 days, and 5 hours for the following reezen: spamming the server and annoying people: also this should be added for bans from channel too, not only server bans: tt vertion: 5.11

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [X] TeamTalkServer

**Comment by bear101** [comment]
Some of these ideas are already in the issues list, e.g. #1373

**Comment by amirmahdifard** [comment]
@bear101 yes there is, but you are not adding them, please add them, please! they are needed for administrators of a tt server

**Comment by amirmahdifard** [comment]
@bear101 bro, if you add this ban duration for banning from server and channel, also ability to type a reezen, so when the user etempted to log in to the server, in the banned from server error they also get they bantime and the reezen, i can give you usd since this feecher is huge, and i need it: thank you.

**Comment by amirmahdifard** [comment]
USD*

**Comment by amirmahdifard** [comment]
@CoBC bro, i have an idea: since you like to work on qt client, you could work on ban duration feecher the client side, and @bear101 can work on server side, i will donate some money to teamtalk, since the app that you've made is really grate, it can do everything want in havving a conversation with someone

**Comment by CoBC** [comment]
@amirmahdifard When @bear101 will be ready to work on it, if it requires my work especially on client side, I'll do it, for now just wait, maybe some other issues have to be treated before this one.

**Comment by amirmahdifard** [comment]
@CoBC @bear101 ok, anyway, rememver that teamtalk is grate, it's the best, i always used to have all my conversation in the teamtalk, and now, for my tt server, i need ban duration, and i'm sure that this will be useful for others too:  because they don't ban people permenantly, so, please do work on this at some point, i will be grateful supporting you guys by donating to teamtalk

**Comment by amirmahdifard** [comment]
i mean, of corce stil allow us to ban permenantly too, but add ban duration so the server will unban in a selected time also: in the ban a user dialogue, add radio buttons, one of them could be permenantly ban, and the other is temerary ban: if the administrator selected the second option, give him ability to type a reezen, and ability for select a bantime, but if the administrator selected the permenant ban option, then only give him an edit box to type a reezen, and there is no bajtime anymore
